ui/src/apiContext.ts

```diff
@@ -4,12 +4,139 @@ import * as tanagra from "./tanagra-api";
 // TODO(tjennison): Figure out a more comprehensive solutions for faking APIs.
 class FakeUnderlaysApi {
   async listUnderlays(): Promise<tanagra.ListUnderlaysResponse> {
+    const columns = [
```

Comment:
This should be a minimal configuration that satisfies the tests. You can check exactly what the tests need but I would expect something like 2 columns and one of each criteria type.